tests/apitest/src/Foundation/NSString.cs

```diff
@@ -54,6 +54,23 @@ public void NSString_BoundingRectWithSize ()
 			Assert.IsTrue (rect.Width > 0);
 			Assert.IsTrue (rect.Height > 0);
 		}
+
+		[Test]
+		public void NSString_CompareTo ()
+		{
+			using (var a = new NSString ("a"))
+			using (var b = new NSString ("b"))
+			using (var c = new NSString ("c"))
+			{
+				var tests = new NSString[] { c, a, b };
+
+				Array.Sort (tests);
+
+				Assert.AreSame (a, tests[0]);
+				Assert.AreSame (b, tests[1]);
+				Assert.AreSame (c, tests[2]);
```

Comment:
please add a string parameter that makes all assert unique (no need to be useful, `, "1"` `, "2"` and `, "3"`
it can fails/crash in some configuration where debug symbols might not be present (or might be broken due to other issues)

Reply:
Done.